packages/synapse-react-client/src/style/components/_goalsv2.scss

```diff
@@ -0,0 +1,123 @@
+@use '../abstracts/variables' as SRC;
+@use 'sass:map';
+
+// have space between items
+%space-between {
+  display: flex;
+  justify-content: space-between;
+}
+// horizontally and vertically center
+%hv-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+%side-padding {
+  padding-left: 25px;
+  padding-right: 25px;
+}
+
+.GoalsV2 {
```

Comment:
There is a lot of duplicate styling between GoalsV1 and GoalsV2, and it's a bit confusing because there is a lot of styling happening both here and in the component file. Can you either:
- Refactor to share styles with GoalsV1 and minimize the duplicates, and put style overrides here instead of in the component
- Remove most/all of these styles and for GoalsV2 just use MUI builtin components and sx styles

Reply:
I deleted the goalsV2 scss file and instead used the original goals styling with some MUI overrides.